src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -623,7 +623,7 @@ void AirspeedModule::select_airspeed_and_publish()
 			 * @description Previously selected sensor index: {1}, current sensor index: {2}.
 			 */
 			events::send<uint8_t, uint8_t>(events::ID("airspeed_selector_estimation_regain"), events::Log::Info,
-						       "Airspeed sensor healthy, start using again", _prev_airspeed_index,
+						       "Airspeed sensor healthy, start using again ({1}, {2})", _prev_airspeed_index,
```

Comment:
I know you're just duplicating the mavlink message, but while we're here can we do something minor to make this more readable?

Something like "Airspeed sensor {1} healthy, switching from ${2}" or "Switching from airspeed ${1} -> ${2} (now healthy)".

@sfuhrer

Reply:
I agree but see now that the description of the event then already is more clear.